agency/README.md

```diff
@@ -253,29 +288,37 @@ If `stop_id` is specified, `GET` will return an array with a single stop record,
 
 [Top][toc]
 
-## Trips
+## Reports
 
-The Trips endpoint serves two purposes: 
+Reports are information that providers can send back to agencies containing aggregated data that is not contained within other MDS endpoints, like counts of special groups of riders. These supplemental reports are not a substitute for other MDS Provider endpoints.
 
-* Definitively indicating that a Trip (a sequence of events linked by a trip_id) has been completed. For example, from analyzing only the raw Vehicle Events feed, if a trip crosses an Agency's jurisdictional boundaries but does not end within the jurisdiction (last event_type seen is a `leave_jurisdiction`), this can result in a 'dangling trip'. The Trips endpoint satisfies this concern, by acting as a final indication that a trip has been finished, even if it ends outside of jurisdictional boundaries; if a trip has intersected an Agency's jurisdictional boundaries at all during a trip, it is expected that a Provider will send a Trip payload to the Agency following the trip's completion.
-* Providing information to an Agency regarding an entire trip, without extending any of the Vehicle Event payloads, or changing any requirements on when Vehicle Events should be sent.
+The authenticated reports are monthly, historic flat files that may be pre-generated by the provider. 
 
-**Endpoint:** `/trips`  
+[Top][toc]
+
+## Reports - register
+
+The `/reports` endpoint allows an agency to register aggregated report counts in CSV structure.
```

Comment:
@marie-x I added Reports to Agency. Check this out for any errors and that it makes sense to you technically.

Reply:
looks good!